distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -64,21 +83,20 @@ Non-Terminating authentication types are authentication types where, once hit, m
 If the client supports the non-terminating authentication type's method of obtaining credentials, it either allows or forbids access to the system.
 However if the client does not support the non-terminating authentication type's method of obtaining credentials, the system will continue to the next configured authentication type.
 
-`BASIC` is the only terminating authentication type.
-Every other authentication type is non-terminating.
+`PKI` is the only non-terminating authentication type.
+`BASIC`, `OIDC`, and `SAML` are terminating authentication types.
```

Comment:
❓ Did you change this, or were the docs just wrong?

Reply:
The docs were incorrect. For example, if you setup OIDC|BASIC as your authentication type and you don't provide OIDC credentials, you won't get the chance to enter basic credentials